curvallis/curve_editing/curve_fitters.py

```diff
@@ -378,11 +388,11 @@ def guess_coefficients(self, points):
             (not hasattr (self, 'rho0'))):
 
             # If not, set defaults for required coefficients
-            if (hasattr (self, 'k0')):
+            if (not hasattr (self, 'k0')):
                 self.k0 = 1.0e11
-            if (hasattr (self, 'k0_prime')):
+            if (not hasattr (self, 'k0_prime')):
                 self.k0_prime = 20.0
-            if (hasattr (self, 'rho0')):
+            if (not hasattr (self, 'rho0')):
```

Comment:
These three lines of code seem to have been written incorrectly according to the comments. Fixing this code brings it into accordance with the comments and allows the SandiaPC fitter to function appropriately without an external part of code changing the user-given rho0 value.